main.go

```diff
@@ -72,24 +93,29 @@ func init() {
 		os.Exit(1)
 	}
 
-	if keyID == "" {
-		keyID = os.Getenv("AWS_ACCESS_KEY_ID")
+	if keyID != "" {
+		os.Setenv("AWS_ACCESS_KEY_ID",keyID)
 	}
 
-	if secretKey == "" {
-		secretKey = os.Getenv("AWS_SECRET_KEY")
+	if secretKey != "" {
+		os.Setenv("AWS_SECRET_ACCESS_KEY",secretKey)
 	}
 
-	if keyID == "" || secretKey == "" {
-		flag.PrintDefaults()
-		os.Exit(1)
+	if region != "" {
+		os.Setenv("AWS_REGION",region)
 	}
 
 	if date == "" {
 		yesterday := time.Now().AddDate(0, 0, -1)
 		date = yesterday.Format("2006-01-02")
 	}
 
+	if influxUrl != "" {
+		fmt.Println("Output influx selected")
```

Comment:
Could you use log instead of println?

Reply:
I'm not sure to understand correctly, I've replaced `fmt` with `log` but maybe I should use debugLogger.